client/components/search-themes/index.tsx

```diff
@@ -83,7 +90,14 @@ const SearchThemes: React.FC< SearchThemesProps > = ( { query, onSearch, recordT
 			updateInput( updatedInput + suggestion );
 			focusOnInput();
 		} else {
-			updateInput( insertSuggestion( suggestion, searchInput, cursorPosition ) );
+			updatedInput = insertSuggestion( suggestion, searchInput, cursorPosition );
```

Comment:
Should we remove these changes, as they are part of another PR https://github.com/Automattic/wp-calypso/pull/70283 ?

Reply:
I should rebase it to trunk

Reply:
The [#70283](https://github.com/Automattic/wp-calypso/pull/70283) has been merged to trunk, so you won't see these changes anymore